### ISSUE TITLE
The case of the missing DEBUG 

### ISSUE THREAD

**Issue by webframp** [spox/batali#89]
As discussed in irc, seen with batali 0.4.10 on Ubuntu 14.04 and Windows 10

```
$ export DEBUG=true
$ batali install
[Batali]: Readying installation destination... complete!
[Batali]: Installing cookbooks... complete!
```

and on Windows:

```
$ $env:DEBUG="true"
$ batali install
[Batali]: Readying installation destination... complete!
[Batali]: Installing cookbooks... complete!
```

**Comment by webframp** [comment]
Overrode the Gemfile to test out the changes,

Gemfile:
```rb
gem 'bogo-cli', git: 'https://github.com/spox/bogo-cli', ref: 'fix/ui-config'
...
```
bundle install:
```
Using bogo-cli 0.2.9 (was 0.2.8) from https://github.com/spox/bogo-cli (at fix/ui-@04f520a)
```
same commands as earlier.

Just to clarify what I think I would see:

https://github.com/spox/batali/blob/master/lib/batali/command/install.rb#L26

seems like it should report unit resolution and I should see the dep graph being walked when debug is enabled via `$env:DEBUG='true'`

so far testing with changes from spox/bogo-cli#10 I haven't seen that but let me know any other info I can provide.

**Comment by webframp** [comment]
Should have checked here first. I knew this seemed familiar.

Digging around here didn't help me too much: https://github.com/spox/bogo-ui/blob/master/lib/bogo-ui/ui.rb#L113

**Comment by webframp** [comment]
Can confirm from testing this is fixed when pointing at: https://github.com/spox/bogo-cli/tree/develop

**Comment by chrisroberts** [comment]
Fixed via #91. Thanks for confirmation @webframp !
